src/components/@shared/FormFields/FilesInput/index.tsx

```diff
@@ -6,12 +6,19 @@ import { InputProps } from '@shared/FormInput'
 import { getFileUrlInfo } from '@utils/provider'
 import { FormPublishData } from 'src/components/Publish/_types'
 import { LoggerInstance } from '@oceanprotocol/lib'
+import content from '../../../../../content/publish/form.json'
+import FormHelp from '@shared/FormInput/Help'
 
 export default function FilesInput(props: InputProps): ReactElement {
   const [field, meta, helpers] = useField(props.name)
   const [isLoading, setIsLoading] = useState(false)
   const { values, setFieldError } = useFormikContext<FormPublishData>()
 
+  const helperMessage = content.services.fields[2].help.replace(
+    'Please enter the URL to your data set file and click "ADD FILE" to validate the data.',
```

Comment:
what are you trying to do here? Why don't you change the actual content instead of clearing it out here?

There should be only 1 help text, not a help tooltip and a help text. So we need a mechanism to have a defined help text either as a tooltip (default) or output directly in field as `FormHelp`